dev-aws/customer-support/topics.tf

```diff
@@ -0,0 +1,15 @@
+resource "kafka_topic" "vulnerability_v7" {
+  config = {
+    # Recommended by dev-ena
+    "compression.type" = "zstd"
+    # Use tiered storage
+    "remote.storage.enable" = "true"
```

Comment:
I see this is created in the team cluster. In there tiered storage is not available.

Reply:
What is the 'team' cluster? I've been trying to string the available docs together to make sense of everything but it's a little tricky

Reply:
"broker-0.broker.customer-support.svc.cluster.aws:9092"
This is a team cluster, hosted in the `customer-support` namespace.
Remote storage and the associated properties `remote.storage.enable` and `local.retention.ms` are only available from the [AWS MSK cluster](https://wiki.uw.systems/posts/shared-kafka-cluster-9ivqpcap) .
We can have a chat if you think it would be helpful.

Reply:
Thanks, that'll be helpful :joy: